data/successors/successor weapons.txt

```diff
@@ -0,0 +1,343 @@
+
+outfit `Bimodal Coilgun Turret`
+	category "Turrets"
+	cost 2950000
+	thumbnail "outfit/bimodal coilgun turret"
+	licenses
+		"High Houses"
+	"mass" 55
+	"outfit space" -55
+	"weapon capacity" -55
+	"turret mounts" -1
+	weapon
+		"hardpoint sprite" "hardpoint/bimodal turret"
+		"hardpoint offset" 9.
+		sprite "projectile/bimodal"
+			"frame rate" 5
+			"start frame" 0
+			"no repeat"
+		sound "bimodal"
+		"hit effect" "seeker impact"
+		"hit effect" "electron impact" 3
+		"fire effect" "moonbeam fleck" 4
+		"inaccuracy" 1
+		"velocity" 30
+		"lifetime" 15
+		"range override" 900
+		"turret turn" 3.5
+		"burst count" 24
+		"burst reload" 3
+		"reload" 15
+		"firing energy" 80
+		"firing heat" 120
+		"firing force" 25
+		"shield damage" -94
+		"hull damage" 75
+		"hit force" 120
+		"cluster"
+		"submunition" "Bimodal Switch" 3
+	description `Successor warships have traditionally relied on fixed weaponry to deal with their foes, but the High Houses do produce a turreted platform for the "Ojet" Coilgun so that their larger and slower vessels can maintain their combat effectiveness against more manuverable targets. It mounts two of the weapons side by side, rigged to fire double the amount of shots per burst to keep them in sync with the fixed hardpoints.`
+
+outfit "Bimodal Switch"
+	weapon
+		sprite "projectile/bimodal switch"
+			"random start frame"
+			"frame rate" 20
+		"hit effect" "ion rain impact" 3
+		"inaccuracy" 10
+		"velocity" 30
+		"lifetime" 6
+		"random lifetime" 3
+		"shield damage" 48
+		"hull damage" 20
+
+outfit "Drag Cannon"
+	category "Guns"
+	cost 455000
+	thumbnail "outfit/drag cannon"
+	licenses
+		"High Houses"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sound "drag cannon"
+		"fire effect" "moonbeam fleck" 2
+		"hit effect" "drag cannon impact" 3
+		"velocity" 0
+		"lifetime" 0
+		"range override" 375
+		"velocity override" 25
+		"burst count" 6
+		"burst reload" 6
+		"reload" 45
+		"firing energy" 30
+		"firing heat" 55
+		"cluster"
+		"submunition" "Drag Projectile" 3
+	description `The Successors have a historical affinity for projectile weaponry which persists to the modern day. The Drag Cannon fires a short-ranged, highly inaccurate blast of projectiles that not only deal substantial damage but also temporarily disrupt the target's engines.`
+	description `While it is an inelegant design in comparison to the time-tested "Ojet" Coilgun, and consequently less appreciated, the Drag Cannon has found a place on the Successors' fast attack craft primarily as a tool for slowing down enemy vessels so that they can be engaged by a larger fleet.`
```

Comment:
```suggestion
	description `	While it is an inelegant design in comparison to the time-tested "Ojet" Coilgun, and consequently less appreciated, the Drag Cannon has found a place on the Successors' fast attack craft primarily as a tool for slowing down enemy vessels so that they can be engaged by a larger fleet.`
```